blog/README.md

```diff
@@ -16,6 +16,17 @@ Note you may need to open the "Screen Options" dialogue at the top of the page t
 
 Similarly, don't forget the "Featured Image" field in the sidebar. If you set an image here, it will be used for social media. This image is not displayed in the post content, however, so if you want to use it as a big image at the top of your post, you'll have to do that AND set the featured image.
 
+## Categories and Tags
+
+Wordpress has a couple tools for organizing your posts: categories and tags. We primarily use categories, and rarely use tags. Here's some background context from Tyler:
+
+>Prior to the current iteration of cloudfour.com, categories and tags were used somewhat inconsistently. While there was some internal understanding of what constituted a tag versus a category, the resultant experience from a user's point of view was just a short, pleasant list of links and then a big ol' pile of random links.
```

Comment:
I feel like you could kill this paragraph. It's interesting background information, but not particularly relevant to the question of "should I use tags and categories?"

Then you can rephrase the opening of the next paragraph to "In the current interation of cloudfour.com, we simplified…"

Reply:
(not trying to nitpick here, I just want to make sure the guide doesn't turn into an overwhelming wall of text)

Reply:
Yeah, that makes sense to me. I'll do that now.

Reply:
Done!

Reply:
👍  works for me. I'll leave final approval to Tyler, since they're his words.